projects/media-viewer/src/lib/toolbar/main-toolbar/right-pane/right-pane.component.html

```diff
@@ -25,8 +25,7 @@
     <span data-l10n-id="bookmark_label">Current View</span>
   </a>
   <div class="toolbarButtonSpacer"></div>
-  <button *ngIf="toolbarButtons.showSubToolbar"
-          id="secondaryToolbarToggle" class="toolbarButton" title="Tools" tabindex="0" data-l10n-id="tools"
+  <button id="secondaryToolbarToggle" class="toolbarButton" title="Tools" tabindex="0" data-l10n-id="tools"
```

Comment:
there needs to be a condition so when the other buttons disappear this is placed, otherwise we have duplicated buttons.

Reply:
that's what PO wanted:

pawel-k 11:55 AM
What buttons should go into submenu unfolded by the chevron button?
highlight
draw
rotate left
rotate right
? those buttons disappear when scalling
should chevron button be visible all the time?
Linda Souto Maior 11:59 AM
Ideally
Chevron button is only available when buttons are hidden
It makes sense to me to hide print and download first
and then maybe highlight and draw
it's odd though because highlight and draw appear before zoom and rotate, so from UX point doesn't make 100% sense
but then rotate/zoom are probably more used features by many users
or majority of users
pawel-k 12:01 PM
the sequence of buttons disappearing may be difficult to implement (edited) 
the fact that print button disappears first does not make more room for hightlight button
Linda Souto Maior 12:01 PM
Ah ok
then i suggest we move from right to left
if sequencing isn't possible
and then as less of the toolbar is visible, buttons are hidden from right-to-left
pawel-k 12:03 PM
ok - that's what happening in my branch atm. I would suggest chevron to be there all the time - otherwise some people may miss it when it appears. if it's there then people may try it before they need it...
Lawrie:racing_motorcycle: 12:10 PM
We're doing this more for accessibility to pick up the buttons and our toolbar is tactical so the chevron button could be there all the time?
Linda Souto Maior 12:14 PM
Yep, agreed!

Reply:
i guess is the PO wants it, i cant argue but i think its not nice to have buttons duplicated.